SF iOS/SF iOS/EventDetails/EventDetailsViewController.m

```diff
@@ -157,5 +166,21 @@ - (void)getTravelTimes {
     }];
 }
 
+// MARK: Share
+
+- (void)share {
+    // TODO: Use coffeecoffeecoffee.coffee url.
```

Comment:
@jamescmartinez do we have any concept of a c4 url for a specific event?

Reply:
@ay8s I just tried adding the URL and it takes away twitter as a share option. That's sort of lame. So I guess we either keep it as is or we get James to add the detail view on web

Reply:
We can just slot the URL into the text and fake it that way. 🤷‍♂ Damn Twitter!

Reply:
Yeah that didn't work for me when I tried it. Maybe I did it wrong.